src/Security/BasicAuth.php

```diff
@@ -82,10 +85,12 @@ public static function requireLogin($realm, $permissionCode = null, $tryUsingSes
 
         $member = null;
         if (isset($_SERVER['PHP_AUTH_USER']) && isset($_SERVER['PHP_AUTH_PW'])) {
-            $member = MemberAuthenticator::authenticate(array(
+            $authenticator = Injector::inst()->get(Authenticator::class);
+
+            $member = $authenticator->authenticate([
                 'Email' => $_SERVER['PHP_AUTH_USER'],
```

Comment:
Make sense, although we'll probably want to clarify the keys that `authenticate()` expects. Perhaps "Username" and "Password", which maps to `Member::$unique_identifier_field` within `MemberAuthenticator::authenticate()`?